src/_macosx.m

```diff
@@ -1520,6 +1528,7 @@ - (void)mouseDown:(NSEvent *)event
                  button = 2;
              else
              {
+                 leftMouseDown = true;
                  button = 1;
                  if ([NSCursor currentCursor]==[NSCursor openHandCursor])
                      [[NSCursor closedHandCursor] set];
```

Comment:
```suggestion
                 button = 1;
                 if ([NSCursor currentCursor]==[NSCursor openHandCursor])
                 {
                     leftMouseDown = true;
                     [[NSCursor closedHandCursor] set];
                 }
```
This avoids showing a closed hand when the click happened outside of the draggable object, or before the pan mode became active.

Reply:
Hm... I purposefully put it outside the if-branch actually because I figured if the cursor is down, we want the closed hand regardless of if we have grabbed anything or not. I can see arguments both ways, so is there a standard to follow for that kind of interaction? With this, there is an open-hand even if I have the mouse down indicating I may be able to grab something again, even though that isn't possible.

Reply:
[MacOS documentation](https://support.apple.com/guide/mac-help/mh35695/mac) mentions:
- Open hand: Appears when the mouse pointer is over an item that you can move and adjust within specific bounds [...]
- Closed hand: Appears when you’re moving and adjusting an item within specific bounds [...]

hence my proposed change. To me it also seem that the closed hand should only ever appear if something is actively being dragged. The open hand with mouse down sounds fine - it means you are not currently grabbing anything, but could if you clicked again.

Reply:
OK, I'm sold. Updated with that behavior now.